go/libkb/pseudonym.go

```diff
@@ -30,17 +29,20 @@ type TlfPseudonymInfo struct {
 	ID      [16]byte
 	KeyGen  int
 	HmacKey [32]byte
+
+	// Ignored in requests. Set in server responses. Using untrusted data
+	// during decryption is safe because we don't rely on TLF keys for sender
+	// authenticity. (Note however that senders must not use untrusted keys, or
+	// else they'd lose all privacy.)
+	UntrustedCurrentName string
```

Comment:
Sure thing. I was on the fence about introducing another variant on this struct (there are already a couple), but I agree that "sometimes defined" fields are even worse.